noir-projects/noir-contracts/contracts/pending_note_hashes_contract/src/main.nr

```diff
@@ -154,10 +156,10 @@ contract PendingNoteHashes {
         let owner_balance = storage.balances.at(owner);
 
         let options = NoteGetterOptions::new();
-        let maybe_notes = owner_balance.get_notes(options);
+        let notes = owner_balance.get_notes(options);
 
-        assert(maybe_notes[0].is_none());
-        assert(maybe_notes[1].is_none());
+        // TOOD: review what this test aimed to test, as it's now superfluous: get_notes never returns 0 notes.
+        assert(notes.len() == 0);
```

Comment:
This function is only used to check that owner has no notes. So I would just nuke this assert and TODO here and just rename the function to `assert_zero_balance`. (Or link an issue like above)